src/components/FormSections/PersonalInfo.jsx

```diff
@@ -253,25 +299,26 @@ function PersonalInfo(props){
                                     maxLength={4}
                                     value={props.fieldData.date_of_birth_year}
                                     onInput={props.saveFieldData('date_of_birth_year')}
-                                    onKeyDown={(e) => restrictType(e, 'number')}
+                                    onKeyDown={(e) => {restrictType(e, 'number'), e.target.setCustomValidity('')}}
+                                    onBlur={(e) => toggleError(e, checkDateValues('year'))}
+                                    onInvalid={(e) => e.target.setCustomValidity(' ')}
                                 />
-                                </label>
                             </div>
                         </div>
-                {(parseInt(dobFieldState.required) && handleErrors.dob) &&
                     <span id="dob-error" rol="alert" className='error-text'>
                         {dobField.error_msg}
                     </span>
-                }
                 </Fieldset>
                 </div>
             </Grid>
             )}
 
             {telephoneFieldState && (
                 <Grid tablet={{ col: 5 }}>
-                    <div className={(parseInt(telephoneFieldState.required) && handleErrors.phone_number) ? 'error-container' : ''}>
-                        <Label className="text-bold" htmlFor="phone-number">{phoneNumberField.label}{(telephoneFieldState.required === "1") && <span className='required-text'>*</span>}</Label>
+                    <div className="input-parent">
```

Comment:
onblur error triggering on phone field even when not required.
![Uploading Screenshot 2024-02-05 at 4.04.41 PM.png…]()

Reply:
fixed with pattern requirement using regex to trigger form invalid